cases/era5_2021.yaml

```
@@ -25,3 +26,8 @@ distancetracking: false
 
 event_start_date: '20210713'
 event_end_date: '20210715'
+
+# Settings for downloading the data
+data_freq: 'hourly'
+data_area: [60, -50,30,30] # North, West, South, East. Default: global
```

```suggestion
data_area: [60, -50, 30, 30]  # North, West, South, East. Default: global
```
Perhaps you can document these settings also in docs/config.md ?